nautobot/dcim/models/racks.py

```diff
@@ -517,7 +518,7 @@ def get_power_utilization(self):
         powerfeeds = PowerFeed.objects.filter(rack=self)
         available_power_total = sum(pf.available_power for pf in powerfeeds)
         if not available_power_total:
-            return 0
+            return (0, 0)
```

Comment:
Anywhere you return a tuple, please return a `collections.namedtuple` instead.

See: https://gist.github.com/lucasnad27/e93b8f316259be3299e5#clarify-multiple-return-values-with-named-tuples